src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -1,15 +1,75 @@
 """
-Download genome files from NCBI FTP server
-"""
+PROTOTYPE - Download genome files from NCBI FTP server.
+
+usage: ncbi_downloader.py [-h] --download_file_ext
+                          DOWNLOAD_FILE_EXT
+                          [DOWNLOAD_FILE_EXT ...] --release_ver
+                          {207,202,95,89,86,83,80}
+                          [--root_dir ROOT_DIR]
+                          [--collection COLLECTION]
+                          [--threads THREADS]
+                          [--chuck_size CHUCK_SIZE] [--overwrite]
+
+options:
+  -h, --help            show this help message and exit
+
+required named arguments:
+  --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...]
+                        Download only files that match given
+                        extensions.
+  --release_ver {207,202,95,89,86,83,80}
+                        GTDB release version
+
+optional arguments:
+  --root_dir ROOT_DIR   Root directory. (default: /global/cfs/cdi
+                        rs/kbase/collections/sourcedata)
+  --collection COLLECTION
+                        Collection (default: GTDB)
+  --threads THREADS     Number of threads. (default: half of
+                        system cpu count)
+  --chuck_size CHUCK_SIZE
+                        Number of genomes per thread
```

Comment:
This is confusing - why not just always distribute evenly rather than allowing the user to do something weird?

Reply:
And what happens when chunk_size * threads is < genome_count?

Reply:
:+1: